src/components/common/SafeTokenWidget/index.tsx

```diff
@@ -11,7 +11,7 @@ import Link from 'next/link'
 import { useRouter } from 'next/router'
 import type { UrlObject } from 'url'
 import Track from '../Track'
-import SafeTokenIcon from './safe_token.svg'
+import SafeTokenIcon from '@/public/images/common/safe_token.svg'
```

Comment:
Nit: please name the file "safe-token.svg" to remain consistent with the others.